### ISSUE TITLE
Consider setting test coverage threshold in Travis CI

### ISSUE THREAD

**Issue by yulgolem** [internetee/registry#1536]
ATM we do not have  test coverage threshold set in Travis, and current master coverage level is about 60%.
Maybe we should pump the coverage level and set the ci threshold for this, like in auction app?

@vohmar , @karlerikounapuu your opinion?

**Comment by vohmar** [comment]
agreed, but feel like a lot of work to bump this overage upto a decent level. Any suggestions what the level should be?

**Comment by yulgolem** [comment]
Well, on one hand assuming our goal is not to set restrictions, but to get max coverage... and on other hand there are some functionality that might be hard to test fully without integration (e.g. registrar portal etc), I suppose maybe we might just dedicate, say, a day per couple of weeks just to pump the tests quantity/quality.
And, I suppose, total coverage threshold might be the current 60% per start, 70%-75% in progress up to 90% ideally.

**Comment by karlerikounapuu** [comment]
In overall, I agree with setting higher coverage threshold. But in my opinion it doesn't resolve any issues we have at the moment, most (if not all) of the PR-s we are pushing are  95%+ cov anyways.

To me, it doesn't really feel right to write tests for functionality I'm not 100% familiar with which renders testing pretty useless. Yup, we might be able to push coverage visually, but that isn't a point actually. We can give it a try though.
